src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
To be honest, since ignoring the whole model will ignore the transformation links I don't know how it will impact the simulation. I've talked with Gustavo and he also thinks that it will affect, but he is not sure as well. So if you see a relevant difference between ignore the model and remove the visual/collisions I can try and verify what will happen.